data/help.txt

```diff
@@ -65,9 +68,26 @@ help "lost 7"
+	`There are also many other useful modes the map can be set to. Click on one of the buttons in the lower right corner to learn more.`
+
+help "map advanced ports"
+	`The Ports panel provides detailed information about every planet, station, or other stellar objects you've come across. Most information can be clicked on (like the word "Outfitter", or on the price of a trade good), which will color every system on the star map appropriately.`
+	`For instance, you can click on "(has been visited)" to see which planets you've visited or not, or click on a trade good price to show how prices change throughout the galaxy. If you have a route planned already, this can be combined with a control-click to inspect trade good prices at stops along the way. A key will be shown on the right to let you know what colors mean in any given mode.`
+
+help "map advanced shops"
+	`Both the Shipyard and Outfitter panels give the same information: A catalog of any items you've seen in your career, providing detailed statistics at any time. Items will be in categories on the left side of the screen, which can be collapsed individually, or all at once with shift-click on a category name. You can also use the "Find" button in the bottom right to search by name.`
+	`When selecting an item, the details will appear in the top-right. Ports which sell it will be colored yellow, while ports with the correct shop but not the item will be colored teal. You can compare what you've selected with a second item with shift-click, which will put both details side-by-side.`
+
 help "multiple ships"
 	`Now that you have more than one ship, while landed on a planet you can reorder the ships in your fleet by clicking and dragging them in the shipyard, outfitter, or player info panel. You will use the first ship in the list as your flagship. In the player info panel you can also "park" a ship on a planet if you want to travel somewhere without it coming with you.`
 
+help "multiple ship controls"
+	`When in flight with multiple ships, you have a variety of detailed commands you can give them. A full list can be seen in the Preferences pane, with <Fleet: Hold position> to have your fleet hold position, or <Fleet: Fight my target> to have your fleet focus your target as examples. These orders will apply to all ships by default, or to your selected ones, if any.`
+	`You can select specific escorts by directly clicking on them, or clicking and dragging your mouse to select a group with a box. These groups can be saved with a control group, by setting one with Control+number key, then recalled with that number key. These can also be set in the Player Info panel, accessed with <View player info>.`
+	`Escorts can also be commanded with the mouse. Right Click on empty space will order escorts to fly to and hold a position, while a click on a hostile ship will focus fire, and a click on a friendly target will cause them to follow.`
```

Comment:
```suggestion
	`You are able to give various commands to any escorts that you own. Commands will be given to all selected escorts, or if no escorts are selected then all your escorts will receive the command. Escorts can be selected by either clicking on them in space, clicking the ship icon at the bottom left of the screen, clicking the ship in the player info panel, or by clicking and dragging to select a group of escorts in space.`
	`Escorts can be saved into control groups by pressing Control+a number key. All selected escorts will then be bound to that number key, and pressing that key will select all escorts in the group.`
	`Commands that can be given include pressing <Fleet: Hold position> to have your escorts hold position or <Fleet: Fight my target> to have your escorts focus fire on the selected ship. A full list of escort commands can be found in the preferences panel.`
	`Escorts can also be commanded with the mouse. Right clicking on an empty point in space will order escorts to fly to and hold position at that point, while clicking on a hostile ship will order your escorts to focus fire on it, and clicking on a friendly target will cause them to follow it.`
```

Reply:
![image](https://user-images.githubusercontent.com/4471575/104133649-880a6500-534a-11eb-9328-d5e60229f679.png)

This just barely overflows on my 1366x768 laptop, and while that small resolution shouldn't be sacrosanct, it's a good indicator it's getting too long. And unlike the Ports tutorial below, this doesn't seem to introduce any new information the previous didn't have. What did you want to do with expanding the text? Is there a way we can strike a balance between the two?

Reply:
How does it look if we ditch the examples of commands and just point the player to the preferences and shorten up some other parts?
```suggestion
	`You can give orders to any escorts that you own. Orders will be given to all selected escorts, or if no escorts are selected then all your escorts will receive the order. Escorts can be selected by either clicking on them in space, clicking the ship icon at the bottom left of the screen, clicking the ship in the player info panel, or by clicking and dragging to select a group of escorts in space. The list of escort order keys can be found in the preferences panel.`
	`Escorts can also be ordered with the mouse. Right clicking on an empty point in space will order escorts to fly to and hold position at that point, while clicking on a hostile ship will order your escorts to focus fire on it, and clicking on a friendly target will cause them to follow it.`
	`Escorts can be saved into control groups by pressing Control+number key. All selected escorts will then be bound to that number key, selecting the group when pressed.`
```

Reply:
I personally prefer the flow of "All escorts will follow an order you give, unless you have selected ones, in which case only they will follow it. Then here's how you select them." But either way it looks good, yeah.

As for the player info panel, I've mostly used it as a convenient way of setting control groups when you don't have to worry about picking ships out from a cluster in space. But it's mostly a convenience, yeah, so it can be cut from a tutorial while in flight.

Reply:
![image](https://user-images.githubusercontent.com/4471575/104135196-2c44d980-5354-11eb-8272-8b2d5c38f5d1.png)


Just added a little pre-amble like `multiple ships`, since the "You can give orders to any escorts you own" looked a little abrupt when taking off. Also changed "groups" to "control groups", A) To add another word to get rid of those massive spaces, and B) to point out to people familiar from RTS games that these are similar. Otherwise, looks much better now.